kedro_airflow_k8s/operators/start_mlflow_experiment.py

```diff
@@ -56,6 +103,10 @@ def execute(self, context: Dict):
         :param context: Airflow context
         :return: mlflow experiment run_id
         """
+        auth_token = self.auth_handler.obtain_token()
+        if auth_token:
+            os.environ["MLFLOW_TRACKING_TOKEN"] = auth_token
```

Comment:
As an alternative to fixing setUp/tearDown, consider clearing this env variable in the end of this function execution